cpp/pipelines/add_subscription_bundle_entries.cc

```diff
@@ -65,6 +65,10 @@ MARC::Record GenerateBundleRecord(const std::string &record_id, const std::strin
         record.insertField("BIB", { { 'a', "1" } });
     if (include_churchlaw)
         record.insertField("CAN", { { 'a', "1" } });
+    if (media_type.compare("online_and_print") == 0 or media_type.compare("online") == 0)
```

Comment:
Please just use the == operator and not the std::string compare member function.  It is more readable and we don't use compare anywhere else.